database/migrations/2018_12_26_029019_create_interests_table.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+use App\Models\Account\Account;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use App\Services\Auth\Population\PopulateModulesTable;
+
+class CreateInterestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('interests', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('account_id');
+            $table->unsignedInteger('contact_id');
+            $table->string('name');
+            $table->unique(['contact_id', 'name']);
```

Comment:
We should make sure this works on Postgre somehow as many instances are using it instead of mySQL.

Reply:
TBH I was following how the `contact pets` resource was made. and this is pretty much exactly how it was done. so should I not follow the existing code as an example of how to implement a feature?

Reply:
I'll get rid of that